src/style.css

```diff
@@ -1,3 +1,55 @@
-body {
-  margin: 0;
+.header {
+  max-width: 1200px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 }
+
+.logo {
+  margin: 10px 0 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin-right: 50px;
+}
+
+.nav__item {
+  height: 60px;
+  line-height: 60px;
+   position: relative;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__is-active {
+  font-family: 'Roboto', sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  color: #000;
+  text-transform: uppercase;
+}
+
+.nav__is-active:hover::after {
+  content: '';
+  position: absolute;
+  border-bottom: 4px solid #00ACDC;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
![Screenshot_6](https://github.com/mate-academy/layout_moyo-header/assets/149514552/f53dd7e5-788f-4a34-9d3f-4a5f2d438e6f)
![Screenshot_7](https://github.com/mate-academy/layout_moyo-header/assets/149514552/94b2c438-1fc5-4fd5-991c-9f33a78b8623)